.nvmrc

```diff
@@ -0,0 +1 @@
+22
```

Comment:
We had [a previous discussion about this](https://github.com/dequelabs/axe-core/pull/3580#discussion_r934793403) where having an `nvmrc` file set to a minimum version of node would force people to use this version rather than any newer version if they use an automatic script to set their node version. Is that something to consider still?

Reply:
We don't have `.nvmrc` files in a lot of our open source repos (https://github.com/dequelabs/axe-core-npm, etc.), which is where the question arrises more so than our internal repos. It's just a question about automatic scripts that use the `.nvmrc` file to switch node versions and if we are still on node 22 as a minimum but node 24 is released, an automatic script would force the user to use node 22 instead of 24. I'm just making sure that's not a concern since a user said [it would be problematic] (https://github.com/dequelabs/axe-core/pull/3580#discussion_r938945769), and this would be the first major open source package to use the `.nvmrc` file strategy.

Reply:
That person is wrong though.